bootstrap.sh

```diff
@@ -1,18 +1,47 @@
 #!/bin/bash
 set -e
 
-# set pwd to script location
-cd "$( dirname "$0" )"
+# Test if the script is running on windows
+windows() {
+    [[ -n "$WINDIR" ]];
+}
 
-# enter the test folder 
+# Load symlink script, it sets the current directory to the root of
+# the repository and export its path in a ROOT variable
+source $(dirname $0)/symlink.sh
+
+cd "$ROOT"
+
+# Enter the test folder
 cd ./tests
 
-# we need stubs from mbed-os
+# We need stubs from mbed-os
 if [ -d "mbed-os" ]
 then
     echo "Using existing mbed-os"
 else
-    #git clone https://github.com/ARMmbed/mbed-os.git
+    # git clone https://github.com/ARMmbed/mbed-os.git
     # until it's not merged we use my branch
     git clone --depth 1 https://github.com/ARMmbed/mbed-os.git -b feature-bluetooth-unit-test
-fi
\ No newline at end of file
+fi
+
+# Add symbolic links
+cd "$ROOT"
+symlink "tests/mbed-os" "tests/TESTS/LinkLoss/device/mbed-os"
+symlink "services/LinkLoss" "tests/TESTS/LinkLoss/device/LinkLoss"
+
+# Create virtual environment
+cd ./tests/TESTS
+mkdir venv
+python3 -m virtualenv venv
+cd venv
+
+# Activate virtual environment
+if windows; then
+  source Scripts/activate
+else
+  source bin/activate
+fi
+
+# Install requirements
+pip install -r ../requirements.txt
```

Comment:
You should install requirements from `mbed-os` too.

Reply:
Done: e898c4dcee2eaceb27c7215f5ad19b582566d1ab